src/Moryx/Serialization/EntryConvert/EntryConvert.cs

```diff
@@ -271,12 +271,27 @@ public static Entry EncodeObject(object instance, ICustomSerialization customSer
             var filtered = customSerialization.GetProperties(instance.GetType());
             foreach (var property in filtered)
             {
+                //exclude property of the same type as the object we are encoding
+                var propertyType = property.PropertyType;
+                if (propertyType == instanceType) continue;
```

Comment:
if you do it like this, it's not possible to edit Properties of the same type as the instance in the UI

Reply:
If you don't do this it lead to stack overflow cause you endup with a circular relation. class->property->class->property...etc

Reply:
If you have a suggestion on how to handle the circular relationship, i am listening.

Reply:
This will not prevent recursion anyway. You could still do "A->B->A->B"...